internal/service/resources/serve.go

```diff
@@ -129,8 +125,13 @@ func Serve(config *api.ResourceServerConfig) error {
 	clientFactory := notifier.NewClientFactory(oauthConfig, utils.DefaultBackendTokenFile)
 	resourceNotifier := notifier.NewNotifier(subscriptionsProvider, notificationsProvider, clientFactory)
 
+	hwMgrDataSourceLoader, err := collector.NewHwMgrDataSourceLoader(cloudID, globalCloudID)
```

Comment:
would it possible to update the `make deploy` or maybe add `make deploy-hardware-manager` in the makefile to get the full stack running? Or maybe there's some instructions elsewhere that can linked? 

Ultimately though is there going to be a single point of deployment (e.g something ACM does where one operator deploys other operator) or is user is expected to deploy one at a time?

Reply:
[here](https://github.com/openshift-kni/oran-hwmgr-plugin/blob/926aa2a18752d5a1eed186541a4528d3a85ea8b1/Makefile#L199)

Reply:
Nice! And the manual steps? Would it helpful to just paste them in readme or like DEVELOPING.md for resource server? Really just want something quick and dirty to have env with HW manager + a happy resource.

Reply:
The README in the plugin repo should have the info you need